### PR TITLE
Add support for mangapy syntax to parser

### PR DESCRIPTION
**Pull request by ChristofferGreen**
# Added
- Added: (Parser) Added support for MangaPy's default naming convention to parser: vol_001-1.cbz

### CODE REVIEW

File: API.Tests/Parser/MangaParserTests.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public MangaParserTests(ITestOutputHelper testOutputHelper)
         [InlineData("Akame ga KILL! ZERO v01 (2016) (Digital) (LuCaZ).cbz", "1")]
         [InlineData("v001", "1")]
         [InlineData("Vol 1", "1")]
+        [InlineData("vol_356-1", "356")] // Mangapy syntax
```
Comment (majora2007) [inline API.Tests/Parser/MangaParserTests.cs:29]: What is the -1? Is it a chapter, if so, we should add a case for it.

File: API/Parser/Parser.cs [hunk 2]
```diff
@@ -83,6 +83,11 @@ public static class Parser
                 @"(?<Series>.*)(\b|_|)(S(?<Volume>\d+))",
                 RegexOptions.IgnoreCase | RegexOptions.Compiled,
             RegexTimeout),
+            // vol_001-1.cbz
```
Comment (majora2007) [inline API/Parser/Parser.cs:86]: Put in your comment this is a format for MangaPy itself. So: // vol_001-1.cbz for MangaPy default naming convention

**Comment by majora2007** [comment]
In order for the build to properly parse, your PR needs to use markdown like the other PRs.

For example, this PR's body should be:

# Added
- Added: Added support for MangaPy's default naming convention to parser <Series>-vol_001-1.cbz. 

If there is an issue, make sure you link it on the PR or use Closes ISSUENUMBER or Fixes ISSUENUMBER if it is a defect. 

Once you add chapter support, I'll test and then can merge if all is good.

**Review by majora2007 — CHANGES_REQUESTED** [review]
Please update PR body to standard and add chapter support

File: API.Tests/Parser/MangaParserTests.cs (hunk 1 above)
Comment (ChristofferGreen) [reply API.Tests/Parser/MangaParserTests.cs:29]: -1, -2, -3, etc is a number for corrections, if they find spelling errors and so on they upload a new version and increase that number. In theory, at least, all the manga I get downloaded have -1 though (weird).

**Comment by majora2007** [comment]
Oh interesting, okay then let me pull it down. I'm planning a release
today, so we can include this. Please adjust the PR title and the comment
as requested.

**Pull request merged**